proto/neutron/cron/tx.proto

```diff
@@ -15,10 +16,48 @@ option go_package = "github.com/neutron-org/neutron/v4/x/cron/types";
 service Msg {
   option (cosmos.msg.v1.service) = true;
 
+  // Adds new schedule.
+  rpc AddSchedule(MsgAddSchedule) returns (MsgAddScheduleResponse);
+  // Removes schedule.
+  rpc RemoveSchedule(MsgRemoveSchedule) returns (MsgRemoveScheduleResponse);
+  // Updates the module parameters.
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
   // this line is used by starport scaffolding # proto/tx/rpc
 }
 
+// MsgAddSchedule is the MsgAddSchedule request type.
+message MsgAddSchedule {
+  option (amino.name) = "cron/MsgAddSchedule";
```

Comment:
what's this for? I mean I don't know :)

Reply:
Me neither :D
I saw such option being used everywhere and just copy-pasted.

Reply:
okay this is just for amino encoding, doesn't matter much, let it stay:
https://github.com/cosmos/cosmos-sdk/blob/e8f28bf5db18b8d6b7e0d94b542ce4cf48fed9d6/proto/cosmos/bank/v1beta1/tx.proto#L41